docs/configuration.rst

```diff
@@ -2734,6 +2734,16 @@ Description
     Also search Plurk comments for URLs.
 
 
+extractor.[postmill].save-link-post-body
+------------------------
```

Comment:
```suggestion
extractor.[postmill].save-link-post-body
----------------------------------------
```

`rst2html` complains with "docs/configuration.rst:2738: (WARNING/2) Title underline too short." otherwise